docs/content/en/docs/workflows/ci-cd.md

```diff
@@ -147,42 +153,53 @@ Starting deploy...
+`skaffold render` builds all application images from your artifacts, templates the newly-generated image tags into your Kubernetes manifests (based on your project's deployment configuration), and then prints out the final hydrated manifests to a file or your terminal. This allows you to capture the full, declarative state of your application in configuration rather than actually applying changes to your cluster, and use this configuration in a GitOps pipeline by committing it to a separate Git repository.
 
-Example of running `skaffold render` to render Kubernetes manifests, then sending them directly to `kubectl`:
+`skaffold apply` will consume one or more fully-hydrated Kubernetes manifests, and send them directly to the Kubernetes control plane via `kubectl` to create resources on the target cluster. After creating the resources on your cluster, `skaffold apply` will use Skaffold's built-in health checking to monitor the created resources for readiness. See [resource health checks]({{<relref "/docs/workflows/ci-cd#waiting-for-skaffold-deployments-using-healthcheck">}}) for more information on how Skaffold's resource health checking works.
 
-Running `skaffold render --output render.txt && cat render.txt` outputs:
+*Note: `skaffold apply` will always use `kubectl` to deploy resources to a target cluster, regardless of deployment configuration in the provided skaffold.yaml. A small subset of deploy configuration will be honored when running `skaffold apply`:*
+* deploy.statusCheckDeadlineSeconds
+* deploy.kubeContext
+* deploy.logs.prefix
+* deploy.kubectl.flags
+* deploy.kubectl.defaultNamespace
+* deploy.kustomize.flags
+* deploy.kustomize.defaultNamespace
+
+{{<alert title="Note">}}
+While `skaffold apply` will attempt to honor the deployment configuration mentioned above, if conflicting configuration is detected in a multi-configuration project, `skaffold apply` will not work.
+{{</alert>}}
+
+`skaffold apply` works with any arbitrary Kubernetes YAML, whether it was generated by Skaffold or not, making it an ideal counterpart to `skaffold render`.
+
+### Example: Hydrating Kubernetes resources using `skaffold render`, then sending them to the cluster using `skaffold apply`:
+
+```code
+-> skaffold render --output render.yaml
+```
 ```yaml
+# render.yaml
 apiVersion: v1
 kind: Pod
 metadata:
   name: getting-started
   namespace: default
 spec:
   containers:
-  - image: gcr.io/k8s-skaffold/skaffold-example:v0.41.0-57-gbee90013@sha256:eeffb639f53368c4039b02a4d337bde44e3acc728b309a84353d4857ee95c369
+  - image: skaffold-example:8fd51906a00a028584c9db41765c51672a45263cd564bbd269690a6181fdcd01
```

Comment:
This image name doesn't look … right.  At least, I don't think rendering output for a local cluster is really a typical usecase.

Reply:
updated